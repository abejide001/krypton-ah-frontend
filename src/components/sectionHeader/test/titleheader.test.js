import React from 'react';
import { shallow } from 'enzyme';
import Header from '..';

describe('titleHeader test', () => {
  it('should test for the titleheader component', () => {
    const headerText = 'headertext';
    const wrapper = shallow(<Header value={headerText} />);
    expect(Header).toBeDefined();
    expect(wrapper.find('Header')).toBeDefined();
    expect(wrapper).toMatchSnapshot();
  });
});
