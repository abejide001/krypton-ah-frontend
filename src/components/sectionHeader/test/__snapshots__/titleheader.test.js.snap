// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`titleHeader test should test for the titleheader component 1`] = `
<Header
  as="h3"
  block={true}
>
  <p
    className="titletext"
  />
</Header>
`;
